lmntfy.py

```diff
@@ -1,40 +1,49 @@
+import os
+os.environ["TOKENIZERS_PARALLELISM"]="False"
```

Comment:
When I use the SBERT embedder, if I do not set that environment variable to false I get the following warning:
 
"
huggingface/tokenizers: The current process just got forked, after parallelism has already been used. Disabling parallelism to avoid deadlocks...
To disable this warning, you can either:
	- Avoid using `tokenizers` before the fork if possible
	- Explicitly set the environment variable TOKENIZERS_PARALLELISM=(true | false)
huggingface/tokenizers: The current process just got forked, after parallelism has already been used. Disabling parallelism to avoid deadlocks...
To disable this warning, you can either:
	- Avoid using `tokenizers` before the fork if possible
	- Explicitly set the environment variable TOKENIZERS_PARALLELISM=(true | false)
"
When I tried setting it  to True, the chatbot got stuck at answering the question, so setting it to false seems to both work and remove the warning. It seems that parallelism will lead to deadlocks.